package.json

```diff
@@ -14,8 +14,10 @@
     "deploy-windows": "SET NODE_ENV=production & webpack --config webpack.production.config.js",
     "heroku-postbuild": "webpack -p --config ./webpack.production.config.js --progress"
   },
-    "engines" : { "node" : "6" },
-    "dependencies": {
+  "engines": {
```

Comment:
is this needed and should we update it?

Reply:
Yes this is needed by Heroku. We should be using Node 6 and not Node 8. Some module, I forgot which one, fails with Node 8

Reply:
got it